server/validation/validation.ts

```diff
@@ -30,7 +31,7 @@ export function validateAsync<T>(
     'unique-subcode',
     async (value, allowedBuildingId, request, passes) => {
       if (typeof value === 'string') {
-        if (!isBlank(value)) {
+        if (isCourtOrBuildingCode(value)) {
```

Comment:
we can only validate codes that pass the lookups own validation, hence check size before doing additional check.
Need to find a better way to  short circuit this validation without this additional check